system-variables.md

```diff
@@ -285,6 +285,22 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Default value: 0
 - This variable is used to control whether to enable the cascades planner, which is currently considered experimental.
 
+### `tidb_enable_clustered_index` <span class="version-mark">New in 5.0</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: 1
+- This variable is used to control whether to enable the Clustered Index feature.
+    - This feature is only applicable to newly created tables and has no effect on old tables that have been created.
+    - This feature is only applicable to tables of single column non-integer primary key and multi-column primary key. It does not affect tables without primary keys and tables with single column integer primary keys.
+    - You can execute `select tidb_pk_type from information_schema.tables where table_name ='{table_name}'` to check whether a table uses the Clustered Index feature.
+- After you enable this feature, rows are stored directly on the primary key instead of the rows_id allocated within the system, and use the extra created primary key index to point to row_id.
```

Comment:
```suggestion
- After you enable this feature, rows are stored directly on the primary key instead of on the internally allocated `rows_id` and using the extra primary key index created to point to `row_id`.
```